### PR TITLE
docs: update en readme

### PR DESCRIPTION
**Pull request by zhouyun-zoe**
**What type of PR is this?**

docs

**What this PR does / why we need it**:

update readme translate

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -26,7 +26,7 @@
 ## 开始
 
 - 如果你想基于火币公链技术搭建自己的区块链，可以参考[快速入门](./getting_started.md)。
-- 如果你想了解火币公链的架构设计，可以参考[架构设计](./arch)、[模块设计](./transaction_pool)。
+- 如果你想了解火币公链的架构设计，可以参考[架构设计](./transaction_pool)。
```
Comment (huwenchao) [inline docs/intro.md]: Wrong reference?

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@
 ## 开始
 
 - 如果你想基于火币公链技术搭建自己的区块链，可以参考[快速入门](./getting_started.md)。
-- 如果你想了解火币公链的架构设计，可以参考[架构设计](./arch)、[模块设计](./transaction_pool)。
+- 如果你想了解火币公链的架构设计，可以参考 [架构设计](./transaction_pool)。
```
Comment (huwenchao) [inline README.md]: All references in this part is broken.

File: README_EN.md [hunk 3]
```diff
@@ -11,32 +11,46 @@ In terms of optimization for financial applications, Huobi chain plans to suppor
 
 ## Get Started
 
-- [Install](./docs/getting_started.md)
-- [Documents](./docs/index.md)
+- If you want to build your own blockchain based on Huobi Chain technology, you can refer to Quick [Start](./docs/getting_started.md).
+- If you want to know the architecture design of the Huobi Chain, you can refer to the [architecture design and module design](./transaction_pool).
+- If you want to develop decentralized applications on Huobi Chain, you can refer to the [contract development tutorial](./contract_demo).
```
Comment (huwenchao) [inline README_EN.md]: Refs here are also broken.

File: docs/en-us/README.md [hunk 4]
```diff
@@ -24,27 +24,31 @@ In terms of optimization for financial applications, Huobi chain plans to suppor
 
 ## Get Started
 
-- [Install](./docs/getting_started.md)
+- If you want to build your own blockchain based on Huobi Chain technology, you can refer to Quick [Start](./docs/getting_started.md).
+- If you want to know the architecture design of the Huobi Chain, you can refer to the [architecture design and module design](./transaction_pool).
+- If you want to develop decentralized applications on Huobi Chain, you can refer to the [contract development tutorial](./contract_demo).
```
Comment (huwenchao) [inline docs/en-us/README.md]: So are these.

Comment (zhouyun-zoe) [reply docs/en-us/README.md]: Done

File: README_EN.md (hunk 3 above)
Comment (zhouyun-zoe) [reply README_EN.md]: Done

File: README.md (hunk 2 above)
Comment (zhouyun-zoe) [reply README.md]: Done

File: docs/intro.md (hunk 1 above)
Comment (zhouyun-zoe) [reply docs/intro.md]: Done

**Pull request merged**